### DISCUSSION TITLE
Local dependencies/packages with vite-react and nextjs templates

### DISCUSSION THREAD

**Discussion by davidduncandev** [codesandbox/sandpack#936] [Q&A]
Does anyone know if local dependencies (https://sandpack.codesandbox.io/docs/guides/local-dependencies) works with vite-react and nextjs templates? I have it working with react template, but as soon as I try vite-react, I can no longer seem to pass the bundled local dependency through the files object to create the node_modules.

**Comment by danilowoz** [comment]
It should work. Could you please give an example that didn't work? 

Thanks

**Reply by davidduncandev** [reply]
Yes, I'm probably missing something obvious, but I cloned and ran https://github.com/codesandbox/sandpack-local-dependencies, but when clicking on load sandbox link, the codesandbox can't find the internal ds local dependency. I see the same if switching over to react-vite template also. I was seeing the same issue with my own project although also seeing the issue with the running sandpack instance before even trying to use the sandbox link, but switching back to react template worked ok.
